src/frontend/src/binder/expr/column.rs

```diff
@@ -92,6 +93,17 @@ impl Binder {
                 }
             }
         }
+        // `CTID` is a system column in postgres.
+        // https://www.postgresql.org/docs/current/ddl-system-columns.html
+        //
+        // We return an empty string here to support some tools such as DataGrip.
+        //
+        // FIXME: The type of `CTID` should be `tid`.
+        // FIXME: The `CTID` column should be unique, so literal may break something.
+        // FIXME: At least we should add a notice here.
+        if let ErrorCode::ItemNotFound(_) = err && column_name.eq_ignore_ascii_case("ctid") {
```

Comment:
PostgreSQL only reserves the lowercase `ctid`. When unquoted, both PG and RW converts identifier to lowercase automatically. But when quoted, a column `"ctiD"` is allowed in PG.

Shall we start rejecting all system columns names in 1.1 when users write `create table` / `create source` / `create materialized view`?

Reply:
Looks reasonable, can you submit another PR to do that?